### PR TITLE
FORTECH-102: Fixed focus on dropdown activator after selection

### PR DESCRIPTION
**Pull request by Djspaceg**
### The Issue
After making a selection from a dropdown menu, focus would be lost and unrecoverable.

### The Solution
Upon closing (hiding) the menu, the focus would remain inside the dropdown list menu popup, and since the list container was set to "self-only", spotlight wasn't allowed to leave that container. However, the container was only hidden, not destroyed, so the normal flow couldn't happen, causing Spotlight to just be stuck in an invisible box, like a real-life mime.

From @germboy:
> I added a `Spotlight.focus()`  bit in the dropdown transition component’s `onHide` method that will set focus back to the dropdown button/item when it closes.

I added several other aria props and ID from Sandstone to help with accessibility.

### CODE REVIEW

**Comment by ybsung** [comment]
> I added several other aria props and ID from Sandstone to help with accessibility.

Thanks for supporting the A11y features in Dropdown.
But we might not need the `ariaLabelledBy` prop.
If we have to support it, then the prop name could be `aria-labelledby`.
But there is no title in the Dropdown, so I think, we could remove it.

**Review by ybsung — APPROVED** [review]
LGTM

File: Dropdown/Dropdown.js [hunk 1]
```diff
@@ -256,6 +272,14 @@ const DropdownBase = kind({
  * @public
  */
 const DropdownDecorator = compose(
+	SpotlightContainerDecorator({
+		enterTo: 'default-element',
+		preserveId: true
```
Comment (MikyungKim) [inline Dropdown/Dropdown.js:277]: Would you please explain why this config needed?

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**